### PR TITLE
level of log incl converting for GELF

### PR DESCRIPTION
**Pull request by JohannesKonings**
https://github.com/Goala/abap-log-exporter/issues/20

### CODE REVIEW

File: src/zcl_ale_log_converter_gelf.clas.abap [hunk 1]
```diff
@@ -19,12 +19,19 @@ CLASS zcl_ale_log_converter_gelf IMPLEMENTATION.
 
     LOOP AT logs ASSIGNING FIELD-SYMBOL(<log>).
 
+      "just a guess. need to checked in GELF doku
```
Comment (Goala) [inline src/zcl_ale_log_converter_gelf.clas.abap]: @JohannesKonings I checked log levels [here](https://www.rubydoc.info/github/graylog-labs/gelf-rb/GELF/Levels).
We could adapt and map some of the to the sap log levels.

File: src/zcl_ale_log_reader_bal.clas.abap [hunk 2]
```diff
@@ -58,7 +58,8 @@ CLASS zcl_ale_log_reader_bal IMPLEMENTATION.
         WITH <message>-msgv1 <message>-msgv2 <message>-msgv3 <message>-msgv4
         INTO log_entry.
 
-      logs = VALUE #( BASE logs ( text = log_entry ) ).
+      logs = VALUE #( BASE logs (   level = <message>-msgty
```
Comment (Goala) [inline src/zcl_ale_log_reader_bal.clas.abap]: @JohannesKonings maybe you can delete some additional whitespaces to reduce some lint error :)

File: src/zcl_ale_log_converter_gelf.clas.abap (hunk 1 above)
Comment (Goala) [reply src/zcl_ale_log_converter_gelf.clas.abap]: Or we just keep 1, 2, 3 and skip the rest

Comment (JohannesKonings) [reply src/zcl_ale_log_converter_gelf.clas.abap]: I added a gelf level constant and mapped it to the internal levels. We can extend or remap that later

File: src/zcl_ale_log_reader_bal.clas.abap (hunk 2 above)
Comment (JohannesKonings) [reply src/zcl_ale_log_reader_bal.clas.abap]: spaces removed

**Comment by JohannesKonings** [comment]
@Goala please review again

**Review by Goala — APPROVED** [review]
👍

**Pull request merged**